### ISSUE TITLE
`KeplerTargetPixelFile.interact()` tutorial does not compile

### ISSUE THREAD

**Issue by barentsen** [KeplerGO/lightkurve#120]
Running `make html` in `docs/` fails with error:
```
Notebook error:
TypeError in tutorials/.ipynb_checkpoints/1.05-interact-with-lightcurves-and-tpf-checkpoint.ipynb:
Object of type 'Undefined' is not JSON serializable
Makefile:20: recipe for target 'html' failed
```

It looks like the `KeplerTargetPixelFile.interact()` tutorial does not compile.  @gully can you look into this? (e.g. may have to replace the widget with an animated gif).  I am going to remove the tutorial for now and open a PR to add it back in.

**Comment by barentsen** [comment]
Yes, I deleted the checkout and encounter the same error.  Can you not reproduce it?

**Comment by barentsen** [comment]
This was fixed by merging #121.  Closing!
